pokedex_async_redux/lib/features/pokemon_details/widgets/moves_card.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex_asyn_redux/api/model/moves.dart';
+import 'package:pokedex_asyn_redux/utils/color_constants.dart';
+import 'package:pokedex_asyn_redux/utils/number_constants.dart';
+import 'package:pokedex_asyn_redux/utils/string_constants.dart';
+
+import 'package:pokedex_asyn_redux/widgets/spacing.dart';
+
+class MovesContainer extends StatelessWidget {
+  const MovesContainer({Key? key, required this.move}) : super(key: key);
+
+  final List<Moves> move;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: [
+        const Text(
+          movesLabel,
+          style: TextStyle(fontSize: 20.0, fontWeight: FontWeight.bold),
+        ),
+        const VerticalSpace(height: 8.0),
+        SizedBox(
+          height: 240.0,
+          child: SingleChildScrollView(
```

Comment:
use List view here since pokemon moves are dynamic based on what specific pokemon is opened 🙂